### DISCUSSION TITLE
GitHub Copilot: Solution 0 returned. finish reason: ["stop"]  Found inline suggestions locally

### DISCUSSION THREAD

**Discussion by xiwu13** [community/community#34845] [Copilot Conversations]
I used Copilot very well yesterday, but suddenly it does not return any suggestions. I have tried to reinstall it, and my Copilot icon is activated at the bottom. 

Have no idea how to solve this issue, could someone help me with it? 

![image](https://user-images.githubusercontent.com/114162722/193441212-6de53954-e9da-42ba-9ee0-0a0b03dc7e04.png)

**Comment by davecheney** [comment]
@xiwu13 copilot is installed and working, it just doesn't have anything to suggest for you at that time.

**Reply by xiwu13** [reply]
It still had the same issues for the last three days. I need to use Alt + \ to show the solution next in line. Did I make any wrong operation by accident? How could I solve this issue?

**Reply by davecheney** [reply]
i'm sorry you are having difficulties. Can you please tell me a bit more about what you tried, and what happened. Screenshots would be very useful. Thank you.

**Reply by xiwu13** [reply]
![image](https://user-images.githubusercontent.com/114162722/193972859-d2525edf-1f56-4d19-b317-85da9576e661.png)
I tried to start coding and after the comment line, it did not show any suggestions. Then I use Alt+\ operation in the next line, it shows the suggestion. But in the output window, it still show the suggestions return 0. 
![image](https://user-images.githubusercontent.com/114162722/193972926-2068aae7-a9ba-42fb-ae99-fb4a6926b23e.png)

**Reply by davecheney** [reply]
This might be a client side problem, please try updating vscode and copilot to the latest versions of the extension.

**Reply by xiwu13** [reply]
My vscode is the latest version and also tried to reinstall the copilot a few times. Do I need to reinstall my vscode in this case?

**Reply by davecheney** [reply]
Thank you. I've checked your telemetry and confirmed you're using the latest vscode and copilot extension.

I have escalated this issue to the copilot extension maintainers.

**Reply by xiwu13** [reply]
Thanks for helping. Appreciate to it :)

**Reply by bmuskalla** [reply]
Would you be so kind to try this in a regular python (or any other file) that is not a notebook if you still encounter the same issue?

**Reply by xiwu13** [reply]
> Would you be so kind to try this in a regular python (or any other file) that is not a notebook if you still encounter the same issue?

Yeah, is the same issue for regular python

**Comment by pboudoin** [comment]
Same issue here

**Comment by urbanscribe** [comment]
same issue here
